### ISSUE TITLE
Allennlp Optuna pruner  

### ISSUE THREAD

**Issue by ofersabo** [himkt/optuna-allennlp#6]
Hi there,

Thanks for this awesome package. 

Is there an allennlp Optuna pruner that is ready to be used?
I tried using the original pruner from pytorch_lighting but I couldn't figure out how to report the metric after each epoch to the metric call back model.

**Comment by himkt** [comment]
Hello @ofersabo, I'm really sorry for the late reply.
For now, the pruner function is partially supported.

You can use Optuna pruners such as Hyperband if you write a model in Python script.
https://github.com/optuna/optuna/blob/master/examples/allennlp/allennlp_simple.py

Currently, I have no idea how to use pruners in `AllenNLPExecutor`.
I'd appreciate it very much if you have any suggestions.

**Comment by himkt** [comment]
Pruning in AllenNLPExecutor is now available.
For more information, please see https://guide.allennlp.org/hyperparameter-optimization#6!
